conf/schemas/cloudtrail.json

```diff
@@ -22,26 +22,31 @@
   "cloudtrail:events": {
     "schema": {
       "additionalEventData": {},
+      "addendum": {},
       "apiVersion": "string",
       "awsRegion": "string",
       "errorCode": "string",
       "errorMessage": "string",
+      "eventCategory": "string",
```

Comment:
```
      eventCategory
Shows the event category that is used in LookupEvents calls.

For management events, the value is management.

For data events, the value is data.

For Insights events, the value is insight.

Since: 1.07

Optional: False
```